scripts/install-cni.sh

```diff
@@ -30,6 +30,21 @@ cilium_ready() {
   # Send errors to stdout since they're "expected" errors.
   # This redirection doesn't affect exit status after execution.
   CNI_COMMAND=VERSION /host/home/kubernetes/bin/cilium-cni 2>&1
+
```

Comment:
Still, could you move this outside cilium_ready() to line 119 (before the change)? This is actually a callback of inotify check and it's called repeatedly before cilium-cni binary is written.

Reply:
Done